packages/core/src/browser/keys.ts

```diff
@@ -1,900 +0,0 @@
-/********************************************************************************
```

Comment:
Could we keep it here to avoid breaking changes?

Reply:
As far as I could tell that code is not used a lot outside of `core`. IMO moving it to the new subfolder helps to keep the code organized.

Reply:
I cannot tell anything about extensions and products outside theia repo. For them we’ve agree to go through deprecation policy when it’s possible.

Reply:
We actually rely on such imports to install listeners for the base widgets, so i would expect that custom widgets are importing them, as tree and mini browser widgets do in Theia repo.

You could do something like:
```ts
import { KeyCode, KeyModifier, ... } from './keyboard';
export { KeyCode, KeyModifier };
```
If you want to restructure without breaking.